packages/arb-token-bridge-ui/src/util/bridgeUiConfig.ts

```diff
@@ -96,15 +96,15 @@ export function getBridgeUiConfigForChain(
         networkLogo: '/images/StylusLogo.svg',
         nativeTokenLogo: ETHER_TOKEN_LOGO
       }
-    case ChainId.Xai:
```

Comment:
Another instance where new Orbit chains would have to have their chain IDs added to `ChainId` enum if we wanted to use them here.

I don't think it's worth it, because we only need to hardcode the chain ID in the config. If it's wrong, we will see errors when testing.